nf_core/utils.py

```diff
@@ -965,13 +965,13 @@ def load_tools_config(directory="."):
 
 
 def determine_base_dir(directory="."):
-    base_dir = Path(directory).absolute()
+    base_dir = start_dir = Path(directory).absolute()
     while not get_first_available_path(base_dir, CONFIG_PATHS) and base_dir != base_dir.parent:
         base_dir = base_dir.parent
         config_fn = get_first_available_path(base_dir, CONFIG_PATHS)
         if config_fn:
             break
-    return base_dir
+    return base_dir.relative_to(start_dir)
```

Comment:
```suggestion
    return directory
```
Would this work?

Reply:
No. I think the actual solution to return the same functionality as before is:

```py
return directory if base_dir == start_dir else base_dir
```

But as stated above I think this means logging would be broken in the cases where you don't return `directory`